sourcegraph/sourcegraph
# feat(setup-checklist): add setup wizard checklist steps behind ff

Closes https://github.com/sourcegraph/sourcegraph/issues/55603.

This PR is the first of many related to the new admin onboarding checklist and adds a base for future setup-checklist PRs to be safely merged while still serving the old setup wizard.

## Test plan
- `sg start`
- Open https://sourcegraph.test:3443/setup and check that has two steps
- Add new `setup-checklist` feature flag and set it to `true`
- Open https://sourcegraph.test:3443/setup and check it has three steps and new navbar item appears

## Demo

https://github.com/sourcegraph/sourcegraph/assets/6717049/0a8bc231-7dfa-40b3-a937-9a5c00cbb562
